language/vm/vm-runtime/src/interpreter.rs

```diff
@@ -259,6 +260,18 @@ impl<'txn> Interpreter<'txn> {
                             )
                         })
                         .collect::<VMResult<Vec<_>>>()?;
+                    let type_context = TypeContext::new(current_frame.type_actuals().to_vec());
```

Comment:
so setting up a call frame now computes both the instantiation for the types and the type tags.
That is going to change as soon as we do the loader work and the binary format changes